fhd_core/polarization/stokes_cnv.pro

```diff
@@ -124,8 +124,9 @@ IF type EQ 8 THEN BEGIN ;check if a source list structure is supplied
     sx=sx[s_use]
     sy=sy[s_use]
     p_ind=p_ind[s_use]
-    flux_arr=Ptrarr(n_pol,/allocate)
-    flux_pq=Ptrarr(n_pol,/allocate)
+    if keyword_set(inverse) then n_pol_stokes=4 else n_pol_stokes=n_pol
```

Comment:
You will need a special case for when `n_pol=1`, since on line 67 above `stokes_list` gets set differently then.

Reply:
Maybe I'm missing something, but why do we have line 67? I think we should just remove that.

Reply:
Feel free to test the code with a single pol with that line removed. At one time it was necessary to prevent a crash, but I suspect that some of the improvements to the way we treat full polarization since then may have also fixed that.